src/ocaml_specific.ml

```diff
@@ -1020,4 +1025,15 @@ begin
     ccflag ~lang ~phase ~flag)))
 end;;
 
+begin
+  let ocamlmklib_pflag ?doc_param flag =
+    pflag ["ocamlmklib"] flag ?doc_param
+      (fun param -> S [A ("-"^flag); A param]) in
+  ocamlmklib_pflag "cclib";
+  ocamlmklib_pflag "ccopt";
+  ocamlmklib_pflag "dllpath";
+  ocamlmklib_pflag "rpath";
```

Comment:
I don't think we need both `-dllpath` and `-rpath` since those are aliases. I would say keep only `-rpath`.

Reply:
Thanks for the comment. My idea was that windows users may find "dllpath" more natural, and unix users prefer "rpath". I'll move to rpath-only.

Reply:
Windows doesn't have the concept of rpath.